snapcraft.yaml

```diff
@@ -0,0 +1,139 @@
+            - libfreetype6
+            - libfreeimage3
+            - libzzip-0-13
+            - libxaw7
+            - libxt6
+            - try: [nvidia-cg-dev] # should be in build-packages, but we have no try: there
+            - try: [libcggl]
+        stage:
+            - -usr/share/OGRE/Media/volumeTerrain
+            - -usr/share/OGRE/Media/materials
+            - -usr/share/OGRE/Media/models
+            - -usr/share/OGRE/Media/thumbnails
+            - -usr/share/OGRE/Media/packs/chiropteraDM.pk3
+            - -usr/share/OGRE/Media/packs/fresneldemo.zip
+            - -usr/share/OGRE/Media/packs/DamagedHelmet.zip
+            - -usr/share/OGRE/resources.cfg
+    mygui:
+        plugin: cmake
+        configflags:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DCMAKE_INSTALL_PREFIX=/usr/
+            - -DCMAKE_CXX_FLAGS=-std=c++11
+            - -DMYGUI_BUILD_DEMOS=FALSE
+            - -DMYGUI_BUILD_TOOLS=FALSE
+        source: https://github.com/MyGUI/mygui.git
+        source-tag: master
+        source-depth: 1
+        after: [ogre]
+    caelum:
+        plugin: cmake
+        configflags:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DCMAKE_INSTALL_PREFIX=/usr/
+            - -DCMAKE_CXX_FLAGS=-std=c++11
+            - -DCaelum_BUILD_SAMPLES=FALSE
+            - -DINSTALL_OGRE_PLUGIN=OFF
+        source: https://github.com/RigsOfRods/ogre-caelum.git
+        source-depth: 1
+        after: [ogre]
+    socketw:
+        plugin: cmake
+        source: https://github.com/RigsOfRods/socketw.git
+        configflags:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DCMAKE_INSTALL_PREFIX=/usr/
+        build-packages:
+            - libssl-dev
+        stage-packages:
+            - libssl1.0.0
+    angelscript:
+        plugin: cmake
+        source: https://github.com/AnotherFoxGuy/angelscript.git
+        source-depth: 1
+        configflags:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DCMAKE_INSTALL_PREFIX=/usr/
+    rapidjson:
+        plugin: cmake
+        source: https://github.com/miloyip/rapidjson.git
+        source-depth: 1
+        configflags:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DCMAKE_INSTALL_PREFIX=/usr/
+            - -DINCLUDE_INSTALL_DIR=/usr/include # rapidjson will add "rapidjson" subdirectory
+            - -DRAPIDJSON_BUILD_EXAMPLES=NO
+            - -DRAPIDJSON_BUILD_TESTS=NO
+            - -DRAPIDJSON_BUILD_THIRDPARTY_GTEST=NO
+    ror:
+        plugin: cmake
+        source: https://github.com/paroj/rigs-of-rods.git
+        source-tag: master
+        source-depth: 1
+        configflags:
+            - -DCMAKE_BUILD_TYPE=Release
+            - -DCMAKE_INSTALL_PREFIX=/usr/
+            - -DBUILD_REDIST_FOLDER=TRUE
+        override-build: |
+            snapcraftctl build
+            make zip_and_copy_resources
+            cp -R bin/content $SNAPCRAFT_PART_INSTALL/root/build_rigs-of-rods/parts/ror/build/redist/bin/
```

Comment:
ideally ror should have a proper install target besides `BUILD_REDIST_FOLDER`, so no fixup is needed